examples/lfric/eg17/full_example_netcdf/README.md

```diff
@@ -108,5 +73,12 @@ you want to use:
 export F90=gfortran
 export F90FLAGS="-Wall -g -fcheck=bound $(nf-config --fflags)"
 
-make
-```
\ No newline at end of file
+make compile
+```
+## Running
+The binary ``example`` can be executed without any parameters:
```

Comment:
Please insert a blank line between the header and the text.

Reply:
Done.